webpack.config.js

```diff
@@ -47,11 +47,12 @@ module.exports = function (env, argv) {
         }]
       }, {
         test: /\.(jpg|png|svg|ico|icns)$/,
-        loader: 'url-loader',
-        options: {
-          limit: 200000,
-          esModule: false
-        }
+        type: 'asset',
```

Comment:
Here to remove deprecated `url-loader`, the corresponding config can be found at https://webpack.js.org/guides/asset-modules/#general-asset-type

Reply:
ok, so please play around it with your webviews. If everything is working well, feel free to approve this PR and we can merge it. (I cannot approve it as original author...)